client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -0,0 +1,152 @@
+import { getSelectedDomain } from 'calypso/lib/domains';
+import { hasGSuiteSupportedDomain } from 'calypso/lib/gsuite';
+import withCartKey from 'calypso/my-sites/checkout/with-cart-key';
+import { BillingSelector } from 'calypso/my-sites/email/email-providers-stacked-comparison//billing-selector/billing-selector';
+import GoogleWorkspaceCard from 'calypso/my-sites/email/email-providers-stacked-comparison/provider-cards/google-workspace-card';
+import ProfessionalEmailCard from 'calypso/my-sites/email/email-providers-stacked-comparison/provider-cards/professional-email-card';
+import { errorNotice } from 'calypso/state/notices/actions';
+import { NoticeOptions } from 'calypso/state/notices/types';
+import { getProductBySlug } from 'calypso/state/products-list/selectors';
+import canUserPurchaseGSuite from 'calypso/state/selectors/can-user-purchase-gsuite';
+import { getDomainsWithForwards } from 'calypso/state/selectors/get-email-forwards';
+import { fetchSiteDomains } from 'calypso/state/sites/domains/actions';
+import { getDomainsBySiteId, isRequestingSiteDomains } from 'calypso/state/sites/domains/selectors';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+import type { Site } from 'calypso/reader/list-manage/types';
+
+import './style.scss';
+
+type EmailProvidersStackedComparisonProps = {
+	cartDomainName?: string;
+	comparisonContext: string;
+	currencyCode?: string;
+	currentRoute?: string;
+	domain?: any;
+	domainName?: string;
+	domainsWithForwards?: any[];
+	hasCartDomain?: boolean;
+	isGSuiteSupported?: boolean;
+	productsList?: string[];
+	requestingSiteDomains?: boolean;
+	shoppingCartManager?: any;
+	selectedSite?: Site | null;
+	selectedDomainName: string;
+	source: string;
+	titanMailMonthlyProduct?: any;
+	gSuiteAnnualProduct?: any;
+};
+
+const recordTracksEventAddToCartClick = (
+	comparisonContext: string,
+	validatedMailboxUuids: string[],
+	mailboxesAreValid: boolean,
+	provider: string,
+	source: string,
+	userCanAddEmail: boolean,
+	userCannotAddEmailReason: any
+) => {
+	recordTracksEvent( 'calypso_email_providers_add_click', {
+		context: comparisonContext,
+		mailbox_count: validatedMailboxUuids.length,
+		mailboxes_valid: mailboxesAreValid ? 1 : 0,
+		provider: provider,
+		source,
+		user_can_add_email: userCanAddEmail,
+		user_cannot_add_email_code: userCannotAddEmailReason ? userCannotAddEmailReason.code : '',
+	} );
+};
+
+const EmailProvidersStackedComparison: FunctionComponent< EmailProvidersStackedComparisonProps > = (
+	props
+) => {
+	const translate = useTranslate();
+	const [ billingPeriod, setBillingPeriod ] = useState( translate( 'monthly' ) );
+	const { comparisonContext, isGSuiteSupported, selectedDomainName, selectedSite, source } = props;
+	const onTermTypeChange = ( termLength: string ) => {
+		if ( termLength === 'monthly' ) {
+			setBillingPeriod( translate( 'monthly' ) );
+			return;
+		}
+		setBillingPeriod( translate( 'annually' ) );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( 'Annually', { context: 'Adverb (as in "Pay annually")'} )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/new-email-comparison-billing-selector)